Library/Formula/msmtp.rb

```diff
@@ -12,6 +12,15 @@ class Msmtp < Formula
 
   depends_on "pkg-config" => :build
   depends_on "openssl"
+  depends_on "gnutls" => :optional
+  depends_on "gsasl" => :optional
+  depends_on "libidn" => :optional
+  depends_on "libsecret" => :optional
+
+  option "with-gsasl", "Use GNU SASL authentication library"
+  option "with-libidn", "Build with support for Internationalized Domain Names"
+  option "with-libsecret"
+  option "with-gnutls", "Use GNU Transport Layer Security library"
```

Comment:
i.e. do you use all these yourself? Is there any reason to not enable any of these by default?